### ISSUE TITLE
Please explain steps to do with datasets and training!

### ISSUE THREAD

**Issue by tommyngx** [tusharsangam/Automatic-Knee-Contour-Detection-Using-U-net-Segmentation#1]
Dear @tusharsangam,
Thank you for your sharing, I love your work. I would like to learn from your repo. 
+ I understand the dataset privacy, if I have a knee X-ray image, what should I do to make it the right format for the training?
+ Could you explain the basic steps of training and testing? 
Thank you!

**Comment by tusharsangam** [comment]
@Tommy-Ngx  Thank you for reaching out to me, It good to see that my work is being useful, I regret that I didn't document it well but I will be happy to help, So in the final I use only Normalize & ContrastAdjustment as my final transforms for Unet, Normalize changes the range of the image to -1 to 1, while ContrastAdjustment is not required but I have done it, most of my code is used from [Integrating Spatial Configuration into Heatmap Regression Based {CNNs} for Landmark Localization](https://arxiv.org/abs/1908.00748) so you can review it

Also if you don't have enough data I have generated an augmentation schedule based on SITK library which I found in the same repo as the paper I mentioned, I first replicate or augment the data save the image & label pair names in pickle file & then make a custom dataloader based on pickle file data
This data loader is very basic &  mimics basic functionality that you can find on pytorch's website tutorial

It's been almost 1 year I did this project & so even it's difficult for me to point out the exact so if you need further help we can arrange a zoom meeting & discuss  what you are looking for

**Comment by tommyngx** [comment]
Thank you! I definitely spend more time and contact you if I have any questions.
